File: email-backup.java/src/main/java/demo/Timer.java
package demo;

import java.util.Date;

final public class Timer {

    final private String name;
    final private long start;

    public Timer(String name) {
        this.name = name;
        this.start = new Date().getTime();
        System.out.println(name + " started");
    }

    public void finish() {
        System.out.println(name + " finished in " + seconds() + " second(s)");
    }

    private int seconds() {
        return ((int) (new Date().getTime() - start)) / 1000;
    }
}

File: email-backup.kotlin/src/main/kotlin/demo/Timer.kt
package demo

import java.util.Date

public class Timer(private val name: String) {

    private val start: Long

    init {
        this.start = Date().time
        println(name + " started")
    }

    public fun finish() {
        println(name + " finished in " + seconds() + " second(s)")
    }

    private fun seconds(): Int {
        return ((Date().time - start).toInt()) / 1000
    }
}
